src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -39,17 +39,17 @@ public static int Run(string[] args)
                 CommandOptionType.SingleValue);
 
             var listTestsOption = cmd.Option(
-                "-lt|--listTests",
+                "-t|--list-tests",
```

Comment:
@dasMulli thought that as well, but `-l` is taken for the `--logger` where it seems much more natural.